recipes/freva-rest-server/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "freva-rest-server" %}
+{% set version = "2411.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/FREVA-CLINT/freva-nextgen/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 7227d57d637e67c1204e00e9e2aae6f4169a0df1e004c01f0e58172a354f3e9a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./freva-rest -vv --no-deps --no-build-isolation
```

Comment:
What `./freva-rest` does here?

Reply:
We've decided to create a multi-part repository to streamline unit testing and general development instead of using submodules or other approaches. The repo hosts code for the client lib, some other services and the rest API server. This conda recipe should only install the rest API server, which is located in the `freva-rest` directory.  Therefore I am installing it from here.